### ISSUE TITLE
How can i use this CardStack with an zoomable imageview?

### ISSUE THREAD

**Issue by ngocht143** [wenchaojiang/AndroidSwipeableCardStack#62]
Hi, i have to integrate this with an imageview which has zoomable feature, but it couldn't call your method to discard a card. How can i do to fix it?
Thanks in advance.

**Comment by ngocht143** [comment]
@wenchaojiang: please help me.

**Comment by einverne** [comment]
Are you manually call the `mCardStack.discardTop(1);` function or swipe to discard card?

**Comment by ngocht143** [comment]
@einverne: Yes i call `mCardStack.discardTop(1);`, my issue is that it can't zoom an image (the swipe listener is called instead of zoom listener)
